e-store/e-store/ScenesScreen/Home/SpecialOffersCollectionView/SpecialOffersSlider.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SpecialOffersSlider.swift
+//  e-store
+//
+//  Created by Ahmed Atef on 01/05/2023.
+//
+
+import UIKit
+
+class SpecialOffersSlider: UIView {
+    
+    // MARK: - Outlets
+    
+    @IBOutlet private weak var specialOffersCollectionView: UICollectionView!
+    
+    // MARK: - Properties
+    
+    var dataSource: [SpecialOffersModel] = []
+    
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+        registerCell()
+        setupDataSourceList()
+        setupFeaturedProductCollectionView()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+        registerCell()
+        setupDataSourceList()
+        setupFeaturedProductCollectionView()
+    }
+    
+    // MARK: - Functions
+    
+    private func commonInit() {
+        fromNib(type: SpecialOffersSlider.self)
+    }
+    private func registerCell() {
+        specialOffersCollectionView.register(UINib(nibName: SpecialOffersCollectionViewCell.identifier, bundle: nil)
+                                             , forCellWithReuseIdentifier: SpecialOffersCollectionViewCell.identifier)
+    }
+    
+    func setupDataSourceList() {
+        dataSource = [SpecialOffersModel(productImage: UIImage(named: Asset.airpodsRemovebgPreview.name) ?? UIImage(),
```

Comment:
plz move those mock into view model

Reply:
Mock has images so i will need to import UIKit in view model